The job **`test-various`** failed! Check out the build log: [(web)](https://github.com/rust-lang-ci/rust/actions/runs/6281542874/job/17060047219) [(plain)](https://github.com/rust-lang-ci/rust/commit/64e3b34dc51429827cf768398f4377fe6e6f1d8d/checks/17060047219/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
failures:

---- [codegen] tests/codegen/simd/simd-wide-sum.rs#llvm stdout ----

error in revision `llvm`: verification with 'FileCheck' failed
status: exit status: 1
command: "/checkout/obj/build/x86_64-unknown-linux-gnu/ci-llvm/bin/FileCheck" "--input-file" "/checkout/obj/build/x86_64-unknown-linux-gnu/test/codegen/simd/simd-wide-sum.llvm/simd-wide-sum.ll" "/checkout/tests/codegen/simd/simd-wide-sum.rs" "--allow-unused-prefixes" "--check-prefixes" "CHECK,NONMSVC,llvm" "--dump-input-context" "100"
--- stderr -------------------------------
/checkout/tests/codegen/simd/simd-wide-sum.rs:55:12: error: CHECK: expected string not found in input
/checkout/tests/codegen/simd/simd-wide-sum.rs:55:12: error: CHECK: expected string not found in input
 // CHECK: zext <16 x i8>
           ^
/checkout/obj/build/x86_64-unknown-linux-gnu/test/codegen/simd/simd-wide-sum.llvm/simd-wide-sum.ll:34:43: note: scanning from here
define noundef i16 @wider_reduce_into_iter(ptr noalias nocapture noundef readonly align 16 dereferenceable(16) %x) unnamed_addr #2 personality ptr @rust_eh_personality {
                                          ^
/checkout/obj/build/x86_64-unknown-linux-gnu/test/codegen/simd/simd-wide-sum.llvm/simd-wide-sum.ll:38:7: note: possible intended match here
 %1 = zext <8 x i8> %_4.i.sroa.0.0.vec.extract to <8 x i16>


Input file: /checkout/obj/build/x86_64-unknown-linux-gnu/test/codegen/simd/simd-wide-sum.llvm/simd-wide-sum.ll


-dump-input=help explains the following input dump.
Input was:
<<<<<<
<<<<<<
            1: ; ModuleID = 'simd_wide_sum.b0ed0802d80e75ee-cgu.0' 
            2: source_filename = "simd_wide_sum.b0ed0802d80e75ee-cgu.0" 
            3: target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128" 
            4: target triple = "x86_64-unknown-linux-musl" 
            5:  
            6: ; Function Attrs: mustprogress nofree nosync nounwind nonlazybind willreturn memory(argmem: read) uwtable 
            7: define noundef i16 @wider_reduce_simd(ptr noalias nocapture noundef readonly align 16 dereferenceable(16) %x) unnamed_addr #0 { 
            8: start: 
            9:  %0 = load <16 x i8>, ptr %x, align 16 
           10:  %1 = zext <16 x i8> %0 to <16 x i16> 
           11:  %2 = tail call i16 @llvm.vector.reduce.add.v16i16(<16 x i16> %1) 
           12:  ret i16 %2 
           13: } 
           14:  
           15: ; Function Attrs: mustprogress nofree norecurse nosync nounwind nonlazybind willreturn memory(argmem: read) uwtable 
           16: define noundef i16 @wider_reduce_loop(ptr noalias nocapture noundef readonly align 16 dereferenceable(16) %x) unnamed_addr #1 personality ptr @rust_eh_personality { 
           17: start: 
           18:  %0 = load <16 x i8>, ptr %x, align 16 
           19:  %1 = zext <16 x i8> %0 to <16 x i16> 
           20:  %2 = tail call i16 @llvm.vector.reduce.add.v16i16(<16 x i16> %1) 
           21:  ret i16 %2 
           22: } 
           23:  
           24: ; Function Attrs: mustprogress nofree norecurse nosync nounwind nonlazybind willreturn memory(argmem: read) uwtable 
           25: define noundef i16 @wider_reduce_iter(ptr noalias nocapture noundef readonly align 16 dereferenceable(16) %x) unnamed_addr #1 personality ptr @rust_eh_personality { 
           26: start: 
           27:  %0 = load <16 x i8>, ptr %x, align 16 
           28:  %1 = zext <16 x i8> %0 to <16 x i16> 
           29:  %2 = tail call i16 @llvm.vector.reduce.add.v16i16(<16 x i16> %1) 
           30:  ret i16 %2 
           31: } 
           32:  
           33: ; Function Attrs: nofree nosync nounwind nonlazybind memory(argmem: read) uwtable 
           34: define noundef i16 @wider_reduce_into_iter(ptr noalias nocapture noundef readonly align 16 dereferenceable(16) %x) unnamed_addr #2 personality ptr @rust_eh_personality { 
check:55'0                                               X~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ error: no match found
           35: bb3.i.i.i.i.preheader: 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~
           36:  %0 = load <16 x i8>, ptr %x, align 16 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           37:  %_4.i.sroa.0.0.vec.extract = shufflevector <16 x i8> %0, <16 x i8> poison, <8 x i32> <i32 0, i32 1, i32 2, i32 3, i32 4, i32 5, i32 6, i32 7> 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           38:  %1 = zext <8 x i8> %_4.i.sroa.0.0.vec.extract to <8 x i16> 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
check:55'1           ?                                                      possible intended match
           39:  %_4.i.sroa.0.8.vec.extract = shufflevector <16 x i8> %0, <16 x i8> poison, <8 x i32> <i32 8, i32 9, i32 10, i32 11, i32 12, i32 13, i32 14, i32 15> 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           40:  %2 = zext <8 x i8> %_4.i.sroa.0.8.vec.extract to <8 x i16> 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           41:  %3 = add nuw nsw <8 x i16> %1, %2 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           42:  %4 = tail call i16 @llvm.vector.reduce.add.v8i16(<8 x i16> %3) 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           43:  ret i16 %4 
check:55'0     ~~~~~~~~~~~~
           44: } 
check:55'0     ~~
           45:  
check:55'0     ~
           46: ; Function Attrs: nonlazybind uwtable 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           47: declare noundef i32 @rust_eh_personality(i32 noundef, i32 noundef, i64 noundef, ptr noundef, ptr noundef) unnamed_addr #3 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           48:  
check:55'0     ~
           49: ; Function Attrs: mustprogress nocallback nofree nosync nounwind speculatable willreturn memory(none) 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           50: declare i16 @llvm.vector.reduce.add.v16i16(<16 x i16>) #4 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           51:  
check:55'0     ~
           52: ; Function Attrs: nocallback nofree nosync nounwind speculatable willreturn memory(none) 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           53: declare i16 @llvm.vector.reduce.add.v8i16(<8 x i16>) #5 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           54:  
check:55'0     ~
           55: attributes #0 = { mustprogress nofree nosync nounwind nonlazybind willreturn memory(argmem: read) uwtable "probe-stack"="inline-asm" "target-cpu"="x86-64" } 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           56: attributes #1 = { mustprogress nofree norecurse nosync nounwind nonlazybind willreturn memory(argmem: read) uwtable "probe-stack"="inline-asm" "target-cpu"="x86-64" } 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           57: attributes #2 = { nofree nosync nounwind nonlazybind memory(argmem: read) uwtable "probe-stack"="inline-asm" "target-cpu"="x86-64" } 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           58: attributes #3 = { nonlazybind uwtable "probe-stack"="inline-asm" "target-cpu"="x86-64" } 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           59: attributes #4 = { mustprogress nocallback nofree nosync nounwind speculatable willreturn memory(none) } 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           60: attributes #5 = { nocallback nofree nosync nounwind speculatable willreturn memory(none) } 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           61:  
check:55'0     ~
           62: !llvm.module.flags = !{!0, !1} 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           63: !llvm.ident = !{!2} 
check:55'0     ~~~~~~~~~~~~~~~~~~~~
           64:  
check:55'0     ~
           65: !0 = !{i32 8, !"PIC Level", i32 2} 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           66: !1 = !{i32 2, !"RtLibUseGOT", i32 1} 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           67: !2 = !{!"rustc version 1.74.0-nightly (64e3b34dc 2023-09-23)"} 
check:55'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
------------------------------------------



```

</details>

--- a/tests/codegen/simd/simd-wide-sum.rs
+++ b/tests/codegen/simd/simd-wide-sum.rs
@@ -52,8 +52,6 @@ pub fn wider_reduce_iter(x: Simd<u8, N>) -> u16 {
 #[no_mangle]
 // CHECK-LABEL: @wider_reduce_into_iter
 pub fn wider_reduce_into_iter(x: Simd<u8, N>) -> u16 {
-    // CHECK: zext <16 x i8>
-    // CHECK-SAME: to <16 x i16>
-    // CHECK: call i16 @llvm.vector.reduce.add.v16i16(<16 x i16>
+    // CHECK: call i16 @llvm.vector.reduce.add
     x.to_array().into_iter().map(u16::from).sum()
 }

